src/documentation/print-query-wiki.ts

```diff
@@ -105,6 +106,26 @@ ${
 	}
 });
 
+registerQueryDocumentation('id-map', {
+	name:             'Id-Map Query',
+	type:             'active',
+	shortDescription: 'Returns the id-map of the dataflow graph of the given code.',
+	functionName:     executeIdMapQuery.name,
+	functionFile:     '../queries/catalog/id-map-query/id-map-query-executor.ts',
+	buildExplanation: async(shell: RShell) => {
+		return `
+If you need access to all nodes flowr has identified in the code, you can retrieve them as a list using this query.
```

Comment:
please consider rewriting this into the nodes in the normalized AST as a mapping from ids to nodes.